scripts/gn/generate_vulkan_icd_json.py

```diff
@@ -28,7 +28,7 @@
 import sys
 
 def glob_slash(dirname):
-    """Like regular glob but replaces \ with / in returned paths."""
+    r"""Like regular glob but replaces \ with / in returned paths."""
```

Comment:
```suggestion
    # Like regular glob but replaces \ with / in returned paths.
```

Reply:
Yes but the current code is a doc string, which is an established pattern.
https://peps.python.org/pep-0257/